-- a/SOURCES/openssl-1.1.1-openssl-cnf-fips-mode.patch     1970-01-01 00:00:00.000000000 +0000
+++ a/SOURCES/openssl-1.1.1-openssl-cnf-fips-mode.patch    2020-01-13 15:17:06.850638770 +0000
@@ -0,0 +1,17 @@
+--- openssl-1.1.1c/apps/openssl.cnf.orig	2020-05-07 18:41:48.140115738 +0000
++++ openssl-1.1.1c/apps/openssl.cnf	2020-05-07 18:42:35.721120857 +0000
+@@ -32,11 +32,11 @@
+ 
+ [ ssl_module ]
+ 
+-system_default = crypto_policy
++alg_section = evp_settings
+ 
+-[ crypto_policy ]
++[ evp_settings ]
+ 
+-.include /etc/crypto-policies/back-ends/opensslcnf.config
++fips_mode = true
+ 
+ [ new_oids ]
+ 
